packages/playground/src/components/pull-request/interfaces.ts

```diff
@@ -109,3 +109,42 @@ export interface TypesPrincipalInfo {
   uid?: string
   updated?: number
 }
+
+export interface TypesCheck {
+  created?: number
+  ended?: number
+  id?: number
+  identifier?: string
+  link?: string
+  metadata?: unknown
+  payload?: TypesCheckPayload
+  reported_by?: TypesPrincipalInfo
+  started?: number
+  status?: EnumCheckStatus
+  summary?: string
+  updated?: number
+}
+
+export interface TypeCheckData {
+  bypassable: boolean
+  required: boolean
+  check: TypesCheck
+}
+
+export interface TypesCheckPayload {
+  data?: unknown
+  kind?: EnumCheckPayloadKind
+  version?: string
+}
+export type EnumCheckPayloadKind = '' | 'markdown' | 'pipeline' | 'raw'
+export type EnumCheckStatus = 'error' | 'failure' | 'pending' | 'running' | 'success'
+
+export enum CheckStatus {
```

Comment:
I feel at some point this would be just `PipelineStatus`.

Reply:
should i just use PipelineStatus?